Fix a problem with having two export links.

hansthen opened the pull request.

randyzwitch commented:
Thanks @hansthen! I'll defer to @blackary about the validity of the JS approach, but it seems reasonable to me

hansthen commented:
I fixed the test. The test is still a bit flaky though. I am not very happy how I solved this, because I just added a small sleep. My gut feeling is that there still is an underlying problem because we send `setStreamlitHeight` too early (and twice). @blackary let me know if this is okay.

blackary reviewed (approved):
Approved!

blackary commented:
Yes, I'm not super happy with the test suite currently, it's definitely flakey. But, I also don't have a great alternative suggestion right now, and I think for what the test is doing, adding a sleep is fine.
